examples/batch_processing.ipynb

```diff
@@ -0,0 +1,1148 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "51c7a1f9",
+   "metadata": {},
+   "source": [
+    "# Batch processing with the Batch API\n",
+    "\n",
+    "The new Batch API allows to **create async batch jobs for a lower price and with higher rate limits**.\n",
+    "\n",
+    "Jobs will be completed within 24h, but can be completed faster depending on global usage. \n",
+    "\n",
+    "This notebook covers how to use the Batch API with a practical example.\n",
+    "\n",
+    "As an example, we will caption images using the Amazon furniture dataset, using the `gpt-4-vision-preview` model. \n",
```

Comment:
instead of using this model, could we use the vision capability in gpt-4-turbo-2024-04-09

Reply:
Will change this!